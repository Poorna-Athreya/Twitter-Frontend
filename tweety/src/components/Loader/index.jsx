import React from 'react';

function Loader() {
  return (
    <div data-testid="loaderText" className="loader">
      Loading...
    </div>
  );
}
export default Loader;
