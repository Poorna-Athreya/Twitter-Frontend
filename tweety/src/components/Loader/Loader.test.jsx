import { render, screen } from '@testing-library/react';
import React from 'react';
import Loader from './index';

describe('Loader', () => {
  it('should have the required elements when Loader is rendered', () => {
    render(<Loader />);
    expect(screen.queryByTestId('loaderText')).toBeTruthy();
    expect(screen.getByText('Loading...')).toBeTruthy();
  });
});
