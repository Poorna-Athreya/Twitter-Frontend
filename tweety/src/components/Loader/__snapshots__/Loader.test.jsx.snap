// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should take a snapshot for Loader 1`] = `
<DocumentFragment>
  <div
    class="loader"
    data-testid="loaderText"
  >
    Loading...
  </div>
</DocumentFragment>
`;
